src/js/components/Calendar/Calendar.js

```diff
@@ -194,32 +194,30 @@ const Calendar = forwardRef(
       // greater than a year. If that's the case, calendar should update without
       // animation.
       let diffBoundsAboveYearFlag = false;
```

Comment:
small naming question, could this be just `diffBoundsAboveYear`? Or, what does `Flag` convey?

Reply:
No problem. I'm just bad with names and tend to put 'flag' in every variable that controls something haha.
I'll change that in the next commit.